src/ctx/StateActionContext.js

```diff
@@ -58,7 +58,10 @@ class StateActionContext extends Context {
     aioLogger.debug('getContextValue(%s)', key)
     // on first run load the tokens from the cloud State
     await this.loadTokensOnce()
-    return cloneDeep(this.data[this.keyNames.CONTEXTS][key])
+    return {
+      data: cloneDeep(this.data[this.keyNames.CONTEXTS][key]),
+      local: false
```

Comment:
keeping `local=false` in this PR is fine as it aligns with the current behavior. see [here](https://github.com/adobe/aio-lib-ims/blob/master/src/ctx/Context.js#L87-L89) and [here](https://github.com/adobe/aio-lib-ims/blob/master/README.md?plain=1#L114)

In the future, we may want to store the `local` information when [setting the token in-memory](https://github.com/adobe/aio-lib-ims/blob/9e80bdd89ba88be549a080acff4553e9f6b3fef3/src/ctx/StateActionContext.js#L97) and return it accordingly.

e.g. this could be done by simply storing it in smthg like `this.isLocal[this.keyNames.CONTEXTS][key]`

This + changes made in this PR will allow an in-memory cache option in Runtime actions, for when State can't be used.

Reply:
@moritzraho can we put your comment as a new issue? anything needed from @jsedding before this is resolved?